application/app/src/test/java/uk/nhs/digital/uec/api/integration/fuzzysearch/AuthServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.nhs.digital.uec.api.integration.fuzzysearch;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.net.URL;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import uk.nhs.digital.uec.api.authentication.model.AuthToken;
+import uk.nhs.digital.uec.api.authentication.model.Credential;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class AuthServiceTest {
+
+  @LocalServerPort private int port;
+
+  @Autowired public TestRestTemplate restTemplate;
+
```

Comment:
Same as above

Reply:
formatting looks alright and sonarqube doesn't complain.